src/main/java/seedu/pill/command/DeleteItemCommand.java

```diff
@@ -0,0 +1,36 @@
+package seedu.pill.command;
+
+import seedu.pill.exceptions.PillException;
+import seedu.pill.util.Item;
+import seedu.pill.util.ItemMap;
+import seedu.pill.util.Storage;
+
+import java.time.LocalDate;
+
+public class DeleteItemCommand extends Command {
+    private final String itemName;
+    private final LocalDate expiryDate;
+
+    public DeleteItemCommand(String itemName) {
+        this.itemName = itemName;
+        this.expiryDate = null;
+    }
+
+    public DeleteItemCommand(String itemName, LocalDate expiryDate) {
+        this.itemName = itemName;
+        this.expiryDate = expiryDate;
+    }
+
+    @Override
+    public void execute(ItemMap itemMap, Storage storage) throws PillException {
+        // Looks stupid, but this way I don't handle Optionals in this class
```

Comment:
Maybe you can rephrase this to [write comments targeting other programmers reading the code](https://nus-cs2113-ay2425s1.github.io/website/se-book-adapted/chapters/codeQuality.html#write-to-the-reader)